api/controllers/game/seven/targeted-one-off.js

```diff
@@ -36,11 +36,11 @@ module.exports = function (req, res) {
                   break;
                 }
                 return Promise.reject({
-                  message: "Your opponent's queen prevents you from targeting their other cards",
+                  message: 'game.snackbar.oneOffs.blockedByQueen',
                 });
               default:
                 return Promise.reject({
-                  message: 'You cannot play a targeted one-off when your opponent has more than one Queen',
+                  message: 'game.snackbar.oneOffs.blockedByMultipleQueens',
```

Comment:
These look like the only two consumers of the redundant oneOff version of this text